src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+@import '/src/styles/blocks/variables';
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  background-color: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
```

Comment:
You don't need to specify full opacity in rgba color. You can simply use `border: 1px solid #f3f3f3;`